docs/development.rst

```diff
@@ -5,6 +5,53 @@
 
 This document contains information on developing llbuild.
 
+Build Instructions
+------------------ 
+
+**Building from source on OSX**
+
+* Install latest Xcode and dependencies::
+
+    $ brew install cmake ninja
+
+* Install FileCheck
+
+* Build::
+
+    $ mkdir build && cd build
+    $ cmake -G Ninja -DCMAKE_BUILD_TYPE:=Debug ..
+    $ ninja
+
+* If cmake errors out with : `Failed to locate 'lit' executable (missing: LIT_EXECUTABLE)`::
+
+    $ export PATH=$HOME/Library/Python/2.7/bin/:"$PATH" 
+                  
+    and then run cmake or:
+                  
+    $ env PATH=$HOME/Library/Python/2.7/bin/:"$PATH" cmake -G Ninja -DCMAKE_BUILD_TYPE:=Debug ..
+
+**Building from source on Ubuntu**
+
+* Install dependencies::
+
+    $ sudo apt-get install clang cmake ninja-build sqlite3 python-pip libsqlite3-dev libncurses5-dev
+      
+* Install lit via pip::
+
+    $ sudo pip install lit
+
+* Install FileCheck
```

Comment:
People are ok with including FileCheck in the toolchains, just need to get a patch for it... I started working on it but testing such changes is a real pain.